mLRS/Common/esp-lib/esp-peripherals.h

```diff
@@ -11,6 +11,7 @@
 
 #define GPIO_INLINE_FORCED  IRAM_ATTR static inline __attribute__ ((always_inline))
 
+// note ESP32 break out every pin, hence the gaps
```

Comment:
probably should be 
// note: ESP32 does not break out ...

Reply:
Addressed